error-prone-contrib/README.md

```diff
@@ -199,9 +199,6 @@ be fully closed, there are some ways in which Refaster's scope of utility could
 be extended. The following is a non-exhaustive list of ideas on how to extend
 Refaster's expressiveness:
 
-- Allow more control over _which_ methods are statically imported by
-  `@UseImportPolicy`. Sometimes the `@AfterTemplate` contains more than one
-  static method invocation, and only a subset should be statically imported.
```

Comment:
Changed my mind about removing this, as I just noticed that this is under the heading "Refaster extension ideas". Will revert. (Actually moving this section elsewhere is for another day.)

Reply:
> (Actually moving this section elsewhere is for another day.)

I can pick that up if you tell me where it is suposed to be 😅

Reply:
Haha, not sure. Eventually we should likely have additional sections on the website, but that requires first knowing what we want. (And even then the text should likely be reviewed first, etc.)

But I'll keep an eye out for other possible PRs! :smile:

Reply:
Alright, maybe you'd be interested in this one: #578 :smile:

Reply:
Thank you 🚀